dev/calltype_correctable_stream_gen.go

```diff
@@ -120,7 +120,8 @@ func (c *Configuration) readCorrectableStream(ctx context.Context, a *ReadReques
 		}()
 	}
 
-	replyChan := make(chan internalState, c.n)
+	expected := c.n
```

Comment:
I haven't looked at this PR closely yet.

But this code is not needed? (It does nothing new/change the logic).
Maybe you didn't mean this intentionally since it's a result of the code generation.

And for stream correctables there are no notion of "expected", since the servers can return an arbitrary number of replies.

Reply:
The `expected` variable is technically only needed for the `per_node_arg` option, but it is perhaps easier to just keep it for both cases to avoid yet another `{{if .PerNodeArg}}` conditional in the template code. To understand when it is needed you need to look at the generated `writePerNode` function in `calltype_quorumcall_gen.go`.

Reply:
Yes, my comment was only targeted at the correctables and not in general.